extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -254,7 +252,7 @@ public function execute() {
 			 * (domain), $this->mCityId is unknown (set to false in constructor)
 			 */
 			wfProfileIn( __METHOD__."-domaincache" );
-			$key = WikiFactory::getDomainKey( $this->mServerName );
+			$key = WikiFactory::getDomainKey( $this->mServerName . '/' . $this->langCode );
```

Comment:
For existing wikis without the path, this would invalidate the cache for all wikis when released because of the added `/`, right? I assume that probably won't be a problem (I don't think there's anything heavy that's cached in it), but wanted to check.

Reply:
Yes, we'll have to bump cache version inside this method anyways since we will need full URL instead of just host. Since it's just a simple query I think this is fine.